backend/development.ini.sample

```diff
@@ -190,9 +204,10 @@ session.cookie_on_exception = True
 
 ## Api key auth is disabled by default,
 ## you should set a not-null value to api.key to active it:
+; api.key =
 api.key = %(basic_setup.api_key)s
 ## base url of the backend API, default to same as website.base_url
-; api.base_url = http://localhost:6543
+; api.base_url =
```

Comment:
There is no default value here ?

Reply:
generated default value are showed as empty to don't make user think that default value is fixed one.